tensorflow/python/ops/lookup_ops.py

```diff
@@ -39,8 +39,7 @@
 from tensorflow.python.util.deprecation import deprecated
 
 
-# TODO(yleon): Remove this function.
-@deprecated("2017-03-02", "Use `tf.tables_initializer` instead.")
+@deprecated(None, "Use `tf.tables_initializer` instead.")
```

Comment:
@martinwicke I am not sure that this is a good idea to push the change date indefinitely. What do you think?

Reply:
@drpngx He told me his opinion #12708

Reply:
Thanks @alanyee.